tests/test_pipeline.sh

```diff
@@ -26,12 +38,14 @@ ls ${SCRIPT_DIR}/resources/remap.vcf \
 # Build the expected VCF
 cat << EOT > "${SCRIPT_DIR}/resources/expected_remap.vcf"
 #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
-chr2	98	.	C	G	50	PASS	.
-chr2	1078	.	G	A	50	PASS	.
+chr2	98	.	C	CG	50	PASS	.
+chr2	1078	.	A	G	50	PASS	.
+chr2	1818	.	AAC	T	50	PASS	.
+chr2	2030	.	A	TCC	50	PASS	.
 EOT
 
 # Compare vs the expected VCF
 diff "${SCRIPT_DIR}/resources/expected_remap.vcf" <(grep -v '^##' "${SCRIPT_DIR}/resources/remap.vcf")
 
 # Clean up after the test
-rm -rf work .nextflow* ${SCRIPT_DIR}/resources/*remap.vcf* ${SCRIPT_DIR}/resources/new_genome.fa.* ${SCRIPT_DIR}/resources/genome.fa.fai
+#rm -rf work .nextflow* ${SCRIPT_DIR}/resources/*remap.vcf* ${SCRIPT_DIR}/resources/new_genome.fa.* ${SCRIPT_DIR}/resources/genome.fa.fai
```

Comment:
Just to confirm, is it intentional that the clean up is now disabled?

Reply:
No, it was not intentional. It was just to retain some files in my machine but forgot to switch it back. Changed, thanks